src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -8,7 +8,7 @@
 @Entity
 @Table(name="CARTS")
 public class Cart {
-    private Long id;
+    private long id;
```

Comment:
w konstruktorze dalej jest duży Long, celowo? coś zyskujemy, tracimy? 
w sumie to jakoś mi to nie przeszkadza ale rzuciło się w oczy

Reply:
przeoczyłem go :)